packages/curve-ui-kit/src/shared/ui/SelectableChip.tsx

```diff
@@ -16,15 +16,16 @@ export const SelectableChip = ({
   toggle: () => void
 } & ChipProps) => (
   <Chip
+    size="large"
     clickable
     color={selected ? 'selected' : 'unselected'}
     onDelete={toggle}
     onClick={toggle}
     deleteIcon={
       <CancelIcon
         sx={{
-          transition: `font-size ${TransitionFunction}`,
-          ...(!selected && { fontSize: '0 !important' }), // unfortunately, MUI overrides the font-size very specifically
+          transition: `width ${TransitionFunction}, height ${TransitionFunction}`,
+          ...(!selected && { '&&&': { width: 0, height: 0 } }),
```

Comment:
lol I gave up after two &'s, I guess three did the trick 🤣

Reply:
:dizzy_face: